### ISSUE TITLE
Create, review, and implement a wireframe for a UI

### ISSUE THREAD

**Issue by Ghoublai-Khan** [PineappleStudios/Super-Pong#21]
Adjusted the scope of this task to create, review, and implement a wireframe for the UI.

**Comment by dg1432** [comment]
This should probably be split up into subtasks. Maybe something along the lines of 1) Create, review, and implement a wireframe (anyone have a LucidChart account or similar software?) 2) Add logic for each of the components. This could probably extend into the main menu UI as well.

**Comment by Ghoublai-Khan** [comment]
Agreed, that sounds good to me. I know nothing about UI so I don't know how to break this up lol. I'll change this into that first point

**Comment by dg1432** [comment]
Game UI Wireframe - https://www.lucidchart.com/invitations/accept/d0f0690f-59d7-4c11-861d-32cc204d9bc7

Hopefully you guys should be able to add notes to it. Click Present to just click through the wireframe.
